crates/sncast/src/starknet_commands/account/create.rs

```diff
@@ -85,7 +85,7 @@ pub async fn create(
         .context("Invalid address")?
         .parse()?;
 
-    let mut message = "Account successfully created. Prefund generated address with at least <max_fee> STRK tokens or an equivalent amount of ETH tokens. It is good to send more in the case of higher demand.".to_string();
+    let mut message = "Account successfully created. Prefund generated address with at least <max_fee> STRK tokens. It is good to send more in the case of higher demand.".to_string();
```

Comment:
```suggestion
    let mut message = format!("Account successfully created. Prefund generated address with at least {max_fee} STRK tokens. It is good to send more in case of higher demand.");
```
And actually we should either replace `STRK` with `FRI` in the message or divide this value by `10^18`. It could also be renamed to `estimated_fee` or something like that.

Reply:
Agreed but let me do this as follow-up PR so we don't mix unrelated changes.